src/extensions/default/InAppNotifications/main.js

```diff
@@ -0,0 +1,372 @@
+/*
+ * Copyright (c) 2019 - present Adobe Systems Incorporated. All rights reserved.
```

Comment:
Minor but necessary: You'll need to change this to `Copyright (c) 2019 - present Adobe. All rights reserved.` due to the recent name change for Adobe in all new documents.

Reply:
Done.